fireant/tests/database/test_pickle.py

```diff
@@ -0,0 +1,29 @@
+import pickle
+
+from unittest.case import TestCase
+
+from fireant import (
+    MySQLDatabase,
+    PostgreSQLDatabase,
+    RedshiftDatabase,
+    SnowflakeDatabase,
+    VerticaDatabase,
+)
+
+
+class DatabasePickleTests(TestCase):
+    def test_that_database_inst_can_be_pickled(self):
+        for db_inst in (
+              MySQLDatabase(),
+              PostgreSQLDatabase(),
+              RedshiftDatabase(),
+              SnowflakeDatabase(),
+              VerticaDatabase(),
+        ):
+            with self.subTest('serialization {}'.format(db_inst.__class__.__name__)):
+                dump = pickle.dumps(db_inst, pickle.HIGHEST_PROTOCOL)
+                self.assertIsNotNone(dump, dump)
```

Comment:
Having dump as assertion message is a bit odd for an assertIsNotNone, no? Because if it will be shown, it will always be None.

Reply:
indeed it is! ok, will remove.